code/game/objects/items/weapons/design_disks/ironhammer.dm

```diff
@@ -13,6 +13,7 @@
 		/datum/design/autolathe/misc/taperecorder,
 		/datum/design/autolathe/tool/tacknife,
 		/datum/design/autolathe/sec/beartrap,
+		/datum/design/autolathe/device/landmine,
```

Comment:
I don't know what I think about this

Reply:
think about how many warcrimes can be committed

Reply:
You can probably increase the license price.

Reply:
fatto